polaris.shopify.com/pages/examples/ExampleStyleGuideComponents/AlignBlockComponent/Placeholder.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {Text, Inline} from '@shopify/polaris';
+
+type Align = 'start' | 'end' | 'center';
+
+export interface PlaceholderProps {
+  label?: string;
+  height?: string;
+  width?: string;
+  childAlign?: Align;
+}
+
+export const Placeholder = ({
+  label,
+  height,
+  width,
+  childAlign = 'start',
+}: PlaceholderProps) => {
+  return (
+    <div
+      style={{
+        background: 'var(--surface-example-block)',
+        padding: '14px 8px 14px 8px',
+        height: height ? `${height}px` : '',
+        width: width ? `${width}px` : '',
```

Comment:
```suggestion
        height: height ?? undefined,
        width: width ?? undefined,
```

That way, you can pass pass the prop value as `width="320px"` and not have to interpolate px.